spec/controllers/events_controller_spec.rb

```diff
@@ -217,33 +217,33 @@
     it "should return an pdf with the eating habits of the user" do
       login(:organizer)
       event = Event.create! valid_attributes
-      
+
       user = FactoryGirl.create(:user)
       profile = FactoryGirl.create(:profile, user: user, last_name: "Peter")
-      application_letter = FactoryGirl.create(:application_letter_accepted, 
+      application_letter = FactoryGirl.create(:application_letter_accepted,
         user: user, event: event, vegan: true)
       user = FactoryGirl.create(:user)
       profile = FactoryGirl.create(:profile, user: user, last_name: "Paul")
-      application_letter = FactoryGirl.create(:application_letter_accepted, 
+      application_letter = FactoryGirl.create(:application_letter_accepted,
```

Comment:
not necessarily, but I like to only have the productive changes in the feature PRs in order to be able to focus on those. If people constantly leave trailing whitespace and editors change it forth and back it might be a good idea to have a cleanup PR where you fix a lot of those formatting problems.